CONTRIBUTING.md

```diff
@@ -21,7 +21,9 @@ After cloning the project run: `yarn`.
 
 After that, run `nps prepare.electron`. Every time you add or remove dependencies in electron/package.json, you will need to rerun `nps prepare.electron`.
 
-After this, run `nps dev.up` to start the dev environment. The application will start the process listening on port 4200. The development is done in the browser, but the server uses electron.
+After this, run `nps dev.up` to start the dev environment. The application will start the process listening on port 4200. The development is done in the browser, but the server uses electron. This is the most straight forward way of building new functionality.
```

Comment:
I just added

> This is the most straight forward way of building new functionality.

but honestly, not married to it. My thinking was to call that out explicitly, since there is many ways to run it.

Reply:
This comment was targeted for line 26 of the readme. Sorry I misclicked

Reply:
I may be missing something, that boots electron for me but doesn't open any windows?